### ISSUE TITLE
mysql: generate command error doesn't include enough info to help debug

### ISSUE THREAD

**Issue by jmhodges** [kyleconroy/sqlc#245]
Running `sqlc` in mysql mode gets me the error:

```
$  sqlc generate
# package sqlcmodels
error parsing file: schema failure: syntax error at position 1362
```

But it's not clear what file wasn't parsed correctly.

**Comment by kyleconroy** [comment]
The error output for MySQL should match the PostgreSQL errors

```
# package authors
authors/query.sql:19:7: column "foo" does not exist
```

**Comment by kyleconroy** [comment]
Fixed in a few PRs (#249, #246)
